pkg/tuple/validation.go

```diff
@@ -51,10 +51,18 @@ func (i *RelationNotFoundError) Error() string {
 	return fmt.Sprintf("Relation '%s' not found in type definition '%s' for tuple (%s)", i.Relation, i.TypeName, i.TupleKey.String())
 }
 
+// ValidateUser returns whether the user is valid.  If not, return error
+func ValidateUser(tk *openfgapb.TupleKey) error {
+	if !IsValidUser(tk.GetUser()) {
+		return &InvalidTupleError{Reason: "missing user", TupleKey: tk}
```

Comment:
Can we make this more explicit? It's possible they could have provided user but explicitly provided it with an empty string (""). So I think it'd be better to return something like  `"the 'user' field must be a non-empty string"`